rocAL/rocAL/include/pipeline/commons.h

```diff
@@ -115,6 +115,15 @@ typedef struct
     unsigned x1, y1, x2, y2;
 } RocalROI;
 
+/*! \brief Last Batch Policies
+ *
+ * currently supports following formats
+ */
+enum RocalBatchPolicy {
+    BATCH_FILL = 0,
```

Comment:
Add comments explaining the policy. Is there a need to have two different enums? There's a similar enum in rocal_api_types.h

Reply:
Yes its required to be present in both commons.h and rocal_api_types.h
Added comments to explain